tests/framework/behaviors/AttributeTypecastBehaviorTest.php

```diff
@@ -181,7 +181,7 @@ public function testSkipNotSelectedAttribute()
         $model->save(false);
 
         $model->refresh();
-        $this->assertEquals('58', $model->amount);
+        $this->assertSame(58, $model->amount);
```

Comment:
Another test that looks invalid. The job of `AttributeTypecastBehavior` is ensuring that `amount` is `int`, but tests expects `string`. Am I missing something?